main.js

```diff
@@ -36,11 +37,33 @@ if (auth.BINANCE_API_KEY && auth.BINANCE_API_SECRET) {
 
           const requiredCurrencyPairs = ['BTCUSDT'];
 
+          // The whole point of map is that it returns an array. So if you wanted to use map,
```

Comment:
Yea, this makes total sense. I actually remember you telling me this before haha. I should've remembered.